test/unzip.js

```diff
@@ -0,0 +1,10 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+describe('unzip', function() {
+  it('returns an array of two arrays', function() {
+    var input = [[1, 2], ['pot', 'kettle']];
+    eq(R.unzip(input), [[1, 'pot'], [2, 'kettle']]);
```

Comment:
I think this would be more demonstrative if the list had more than two elements.  The behavior of this example is indistinguishable from `zip`.  Moreover, I would find it more appropriate if the resulting lists are more homogeneous.  Something like:

```js
R.unzip([[1, 'a'], [2, 'b'], [3, 'c'], [4, 'd']]); //=> [[1, 2, 3, 4], ['a', 'b', 'c', 'd']]
```

I suppose the same length comment applies to the code documentation too.